studio/schemas/documents/lesson.js

```diff
@@ -34,6 +34,19 @@ export default {
       name: 'awsFilename',
       type: 'url',
     },
+    {
```

Comment:
Added a resource field to lessons. This field can support multiple types of documents but for now we are only using videoResource